src/meta/processors/partsMan/GetSpaceProcessor.cpp

```diff
@@ -11,17 +11,24 @@ namespace meta {
 
 void GetSpaceProcessor::process(const cpp2::GetSpaceReq& req) {
     folly::SharedMutex::ReadHolder rHolder(LockUtils::spaceLock());
-    auto spaceId = req.get_space_id();
+    auto spaceRet = getSpaceId(req.get_space_name());
+    if (!spaceRet.ok()) {
+        resp_.set_code(to(std::move(spaceRet.status())));
```

Comment:
use `std::move(spaceRet).status()` instead.

BTW. Please change `to` to passing by const reference. Then here `std::move` is no longer needed.

Reply:
Good catch! 👍